src/openrct2/peep/Peep.cpp

```diff
@@ -329,6 +329,11 @@ bool rct_sprite::IsPeep() const
     return peep.sprite_identifier == SPRITE_IDENTIFIER_PEEP;
 }
 
+bool rct_sprite::IsVehicle() const
+{
+    return peep.sprite_identifier == SPRITE_IDENTIFIER_VEHICLE;
+}
```

Comment:
Wouldn't this be better to defined in `src/openrct2/ride/Vehicle.cpp` and use the `vehicle` identifier rather than `peep`?

Reply:
Leaving it as `peep` was an unintentional, nice spot. I didn't actually use these functions in the end, so have removed them.